### ISSUE TITLE
Fatal error: Exit code 1 () Requires the Python library rpy (to call R)

### ISSUE THREAD

**Issue by remimarenco** [peterjc/pico_galaxy#12]
Hi Peter,

I managed to install the tool by hand, but it seems to need rpy...do we need the old rpy to resolve this?

Thanks!

**Comment by peterjc** [comment]
If you are referring to https://github.com/peterjc/pico_galaxy/tree/master/tools/venn_list yes, it needs rpy v1 not rpy2. Like the rest of Galaxy, it really ought to be updated since rpy v1 has been superseded.

Have I misunderstood, or can I retitle this bug for updating ``venn_list`` to use rpy2?

**Comment by peterjc** [comment]
Or did you mean to comment on #11 instead?

**Comment by remimarenco** [comment]
Yes that would be nice to use rpy2! As it is not really an issue, I guess we can rename it, thanks!

**Comment by bgruening** [comment]
@remimarenco is this one not installing for you properly?
I have changed the requirements in https://github.com/peterjc/pico_galaxy/commit/370b7cdcd82c2b5172d15c4f45edcd9bb14d8662 and it should work now.

**Comment by remimarenco** [comment]
Thx @bgruening! Do you think it is possible to update the toolshed repository with this?

**Comment by peterjc** [comment]
I think you're discussing the error on #11, but I thought the main and Test Tool Shed already had @bgruening's update https://github.com/peterjc/pico_galaxy/commit/370b7cdcd82c2b5172d15c4f45edcd9bb14d8662 included:

* https://testtoolshed.g2.bx.psu.edu/view/peterjc/venn_list/c4ec51cb5acc - most recent commit was a readme tweak
* https://toolshed.g2.bx.psu.edu/view/peterjc/venn_list/26e35d5133a1 - aka v0.0.9

**Comment by remimarenco** [comment]
Hmmm, I will try the manual installation and get the requirements through the toolshed!

Thanks for the explanation :). (I did not check the date, thought it was a now-update :p )
